src/ngrest/base/ActiveButton.php

```diff
@@ -154,7 +155,29 @@ public function getCondition()
     {
         return empty($this->_condition) ? '' : $this->_condition;
     }
-
+    
+    private $_permissionLevel = null;

```

Comment:
private $_permissionLevel;

is equals to

private $_permissionLevel = null;

So no need for default value definition.